node/src/command.rs

```diff
@@ -411,9 +420,31 @@ pub fn run_with(cli: Cli) -> Result {
             .into()),
         None => {
             let runner = cli.create_runner(&cli.run.normalize())?;
+            let chain_spec = &runner.config().chain_spec;
+            let is_dev = chain_spec.is_localdev();
+            info!("id:{}", chain_spec.id());
             let collator_options = cli.run.collator_options();
 
             runner.run_node_until_exit(|config| async move {
+                if is_dev {
+                    info!("DEV STANDALONE MODE.");
```

Comment:
Can we give this a big red 🛑 or ⚠️ emoji so it stands out more, the line is a bit buried in the normal logs